pysus/ftp/databases.py

```diff
@@ -106,20 +119,31 @@ def describe(self, file: File) -> dict:
         dis_code, year = self.format(file)
 
         description = dict(
-            name=file.name,
+            name=file.basename,
             disease=self.diseases[dis_code],
-            year=year,
+            year=zfill_year(year),
             size=humanize.naturalsize(file.size),
             last_update=file.date.strftime("%d/%m/%Y %H:%M"),
         )
         return description
 
+    def format(self, file: File) -> tuple:
+        year = file.name[-2:]
+
+        if file.name.startswith("SRC"):
+            dis_code = file.name[:3]
+        elif file.name == "LEIBR22":
+            dis_code = "LEIV"  # MISPELLED FILE NAME
```

Comment:
This is not really a mispelling the V corresponds to _Visceral Leishmaniasis_, but still does not follow the pattern of the other file names, so it is ok to add this special treatment here.